Uni_life_tracker/app/src/main/res/layout/activity_settings_page.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                                  android:layout_width="match_parent"
+                                                   xmlns:app="http://schemas.android.com/apk/res-auto"
+                                                   android:layout_height="match_parent">
+
+    <ImageButton
+            android:id="@+id/arrow_button"
```

Comment:
I think it would help the backend if you could use the id here to get back to the previous page. The code should be very similar to the button from the main activity

Reply:
Ok I'll see if I can add that now

Reply:
I can't add that until we have every other UI done because you return to the main page from settings and the main page is not done rn